modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/refresh/RefreshPodcastsTask.kt

```diff
@@ -103,9 +103,6 @@ class RefreshPodcastsTask @AssistedInject constructor(
                     applicationScope = applicationScope,
                     runNow = true,
                 )
-                if (!refreshThread.isAllowedToRun(runNow = true)) {
-                    return@withContext
-                }
```

Comment:
This isn't needed here as `RefreshPodcastsThread` already handles this scenario.